io/CMakeLists.txt

```diff
@@ -4,19 +4,21 @@
 #
 # Mozilla Public License Version 2.0
 
-# Let the user know what's happening.
-message(STATUS "Building 'detray::io' component")
+if( DETRAY_IO )
```

Comment:
Of course this file should **only** be modified to add `add_subdirectory(common)`, but for the future: You can skip the processing of a subdirectory with:

```cmake
if( NOT DETRAY_IO )
   return()
endif()
```

Instead of putting everything inside of the if-statement.